mappings/net/minecraft/recipe/Ingredient.mapping

```diff
@@ -5,30 +5,32 @@ CLASS net/minecraft/class_1856 net/minecraft/recipe/Ingredient
 	FIELD field_9019 entries [Lnet/minecraft/class_1856$class_1859;
 	METHOD <init> (Ljava/util/stream/Stream;)V
 		ARG 1 entries
-	METHOD method_8086 fromPacket (Lnet/minecraft/class_2540;)Lnet/minecraft/class_1856;
+	METHOD method_26964 create (Ljava/util/stream/Stream;)Lnet/minecraft/class_1856;
+		ARG 0 stream
+	METHOD method_8086 read (Lnet/minecraft/class_2540;)Lnet/minecraft/class_1856;
 		ARG 0 buf
 	METHOD method_8088 write (Lnet/minecraft/class_2540;)V
 		ARG 1 buf
 	METHOD method_8089 toJson ()Lcom/google/gson/JsonElement;
 	METHOD method_8090 (Lcom/google/gson/JsonElement;)Lnet/minecraft/class_1856$class_1859;
 		ARG 0 jsonElement
-	METHOD method_8091 ofItems ([Lnet/minecraft/class_1935;)Lnet/minecraft/class_1856;
+	METHOD method_8091 create ([Lnet/minecraft/class_1935;)Lnet/minecraft/class_1856;
```

Comment:
Don't like this. I'd just do `of` for this and the other similar methods for Tags and Entries